src/main/java/com/microsoft/sqlserver/jdbc/SQLServerError.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
+ * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+
+package com.microsoft.sqlserver.jdbc;
+
+import java.io.Serializable;
+
+/**
+ * StreamError represents a TDS error or message event.
+ */
+

```

Comment:
extra line

Reply:
Thanks for pointing out - will set comments too..